oci/memory_store.go

```diff
@@ -24,18 +24,26 @@ func (c *memoryStore) Add(id string, cont *Container) {
 }
 
 // Get returns a container from the store by id.
-func (c *memoryStore) Get(id string) *Container {
+func (c *memoryStore) Get(idOrName string) *Container {
```

Comment:
Why are we mutating existing functions? I would strongly prefer that we retain Get()'s current semantics.

I also don't feel like functions are appropriate for MemoryStore. Make them act on ContainerServer, so you can use the Name and ID indexes for fast lookups.